src/main/java/racingcar/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String text) {
+
```

Comment:
유틸성 클래스네요
의도하지 않은 인스턴스화를 방지하기 위해서 
private 기본 생성자를 선언해두는 것이 좋습니다.

```suggestion
public class StringAddCalculator {
    private StringAddCalculator { }
    
    public static int splitAndSum(String text) {
```